Tests iOS/Support/Elements/SavesElement.swift

```diff
@@ -48,6 +53,10 @@ struct SavesElement: PocketUIElement {
         element.otherElements[type]
     }
 
+    func searchEmptyStateView(for type: String) -> XCUIElement {
+        element.images[type]
```

Comment:
this boggles my mind, but the accessibility identifier is found through the image type for the `EmptyStateView`